examples/hybrid/sphere/baroclinic_wave_utilities.jl

```diff
@@ -242,8 +242,10 @@ end
 # TODO: Allow ClimaCore to handle both 2D and 3D Fields in a single broadcast.
 #       This currently results in a mismatched spaces error.
 function vertical_diffusion_boundary_layer_cache(
-    Y;
+    Y,
+    ::Type{FT};
```

Comment:
Need to add `where {FT}` after the closing `)`, otherwise this is referencing the globally defined `FT`.